src/log/file_appender.cpp

```diff
@@ -155,6 +155,22 @@ namespace fc {
       }
    }
 
+    file_appender::file_appender( const config& arg ) : fc::appender(), 
+        my( new impl( arg ) )
```

Comment:
Where in the code used this constructor except the new test case?

Reply:
this test case made @jmjatlanta maybe he can help to answer on your question

Reply:
I think nowhere in the code this constructor was used except current test case, so @abitmore do you mean we don't need it ?

Reply:
Sorry guys, I added that constructor as a cruel way to construct a file_appender. I cannot create an fc::variant<config> as it will not compile. Rather than fight that battle, I threw in a new constructor. We now need to fight the battle and win, and then we can eliminate this constructor.

Perhaps I can look at it again with fresh eyes to spot where the problem is. I'll update my results here.

Reply:
:+1:

Reply:
Fixed the variant issue. See this commit: https://github.com/bitshares/bitshares-fc/commit/21cc74468683ea66ffcbced4e18bf3c5ae09d63e

Reply:
:+1:

Reply:
And now fixed the SIGSEGV in the test. See this commit: https://github.com/bitshares/bitshares-fc/commit/a549c6e6db81854031cc5c3e42dc1e65c4d4389c

Reply:
:+1: Thank you very much ! Great job done !